module-2/backend/week_two.md

```diff
@@ -5,33 +5,97 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  * ActiveRecord is an example of Object Relational Mapping. It is an API that facilitates the communication between Ruby/Rails and a SQL database. Most simply, we can make queries to the database in Ruby-y language.
+  
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  * These are instance methods. An instance of a model can call one of these association methods to find its related data. E.G., a school has many students, but a has many method `:students` would apply to a specific school, not the abstract idea of School (~class). The same applies for a student, who belongs to a school at a given time.
```

Comment:
This is a stale version of the questions. Be sure to set the Turing repo up as an [upstream repository](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and then [fetch the latest version](https://help.github.com/articles/syncing-a-fork/) before you start working on the questions for the week.